lib/metrics.py

```diff
@@ -170,7 +171,7 @@ def check_metric(**params):
     minu = metric.get("minu", 1.0)
     minv = metric.get("minv", 1.0)
     ssim = calculate_ssim(
-      params["reference"], params["decoded"],
+      params["reference"], params.get("decoded", params["ofile"]),
```

Comment:
The test should set params["decoded"] instead.

Reply:
I think it is not accurate which all output are named decoded.

Reply:
If you have a better, more "generic" name, then please feel free to propose it.

Reply:
It doesn't make sense for check_metric to handle all possible output names that a test could use.  Thus, a standard/generic name is easier to maintain at this level.